### PR TITLE
Updates contributing guide regarding cosmetic changes

### PR DESCRIPTION
**Pull request by Harsh062**

### CODE REVIEW

File: CONTRIBUTING.md
Comment (rajatkapoor) [inline CONTRIBUTING.md:1]: We can also add a section for community contributed themes and link them in the README like we discussed

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+
+### Theme Contributions
+
+While we appreciate the creativity in creating new themes, we want to keep the starter kit streamlined and focused on its core themes: Personal, Enterprise, and Hashnode. Therefore, we do not accept direct PRs for adding new themes to the main repository. Instead, we encourage you to:
+
+1. **Host your theme in a separate repository**.
+2. **Deploy your theme to a live demo site**.
+3. **Open an issue on our GitHub repository and provide the links in the description.**.
+
+We will showcase these themes in our README file, allowing others to discover and use them.
```
Comment (rajatkapoor) [inline CONTRIBUTING.md:28]:
```suggestion
We will showcase these themes under the `Community Themes` section below, allowing others to discover and use them.
```

**Review by rajatkapoor — COMMENTED** [review]
Left one comment. 
Can we also make the community theme section a table?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Harsh062) [reply CONTRIBUTING.md]: Done

**Review by rajatkapoor — APPROVED** [review]

**Pull request merged**